src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7256353)
```suggestion
import React from "react";
```